### ISSUE TITLE
how to add and showing playlist

### ISSUE THREAD

**Issue by vista136** [pejonic/cordova-pejonic-plugin-exoplayer#7]
i want to add and show playlist in this player... how to ?

**Comment by johnnwanosike1** [comment]
https://medium.com/google-exoplayer/dynamic-playlists-with-exoplayer-6f53e54a56c0

**Comment by vista136** [comment]
how to add with some div and displaying on top this player

**Comment by johnnwanosike1** [comment]
That will mean that you want us to design your app?

**Comment by vista136** [comment]
no, that's not what I mean, I've made the div I gave z-index 999999999 in css, but the div stays behind the player ... how do I do it so that the div can be set on the top  this player

**Comment by johnnwanosike1** [comment]
You will need raw java to do that! Reason is that if you open the player, it takes full screen and therefor hiding other action or events.
